### PR TITLE
Update CHANGELOG.md

### PR DESCRIPTION
**Pull request by vsinghal13**

### CODE REVIEW

**Comment by vsinghal13** [comment]
@seansain Can we make a new release with these changes?

**Comment by seansain** [comment]
Yes, i can do a release today after this is merged

**Comment by seansain** [comment]
You can do Changelog edits in the PR itself next time

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,11 @@
 ## 2.2.0 (Unreleased)
+
+DOCS:
+* Add docs for common source properties ([#50](https://github.com/terraform-providers/terraform-provider-sumologic/pull/50))
```
Comment (seansain) [inline CHANGELOG.md:4]: On the Changelog we edit we write (GH-50), we don't put the link
The hashicorp bot will go and replace these

**Pull request merged**